core/pom.xml

```diff
@@ -167,6 +167,31 @@
 					</dependency>
 				</dependencies>
 			</plugin>
+			<plugin>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
+				<extensions>true</extensions>
+				<executions>
+					<execution>
+						<id>jar</id>
+						<configuration>
+							<bnd><![CDATA[
+				                Bundle-Copyright: Copyright (c) (2019) Contributors to the Eclipse Foundation.
```

Comment:
I have to admit that I've never seen the use of that header?
Can't we just drop it?

Reply:
https://docs.osgi.org/specification/osgi.core/7.0.0/framework.module.html#d0e1831

Sure we can drop

Reply:
removed